draft-ietf-taps-interface.md

```diff
@@ -2673,22 +2679,23 @@ send, it cannot be received after this call.
 Connection.Close()
 ~~~
 
-The Closed Event can inform the application that the Remote Endpoint has closed the
-Connection; however, there is no guarantee that a remote Close will indeed be
+The Closed Event informs the application that the Remote Endpoint has closed the
+Connection. There is no guarantee that a remote Close will indeed be
 signaled.
 
 ~~~
 Connection -> Closed<>
 ~~~
 
-Abort terminates a Connection without delivering remaining data:
+Abort terminates a Connection without delivering any remaining data:
 
 ~~~
 Connection.Abort()
 ~~~
 
-A ConnectionError informs the application that data to could not be delivered after a timeout,
-or the other side has aborted the Connection; however, there is no guarantee that an Abort will indeed be signaled.
+A ConnectionError informs the application that data could not be delivered after a timeout,
+or afer the other side has aborted the Connection. 
```

Comment:
This is a semantic change. If all data are delivered (e.g., consider that the where this error is firing has no more data to send), and then the other side does "Abort", a ConnectionError will still happen.

Suggestion:
A ConnectionError informs the application that one of the following things happened: 1) data could not be delivered after a timeout, or 2) the other side has aborted the Connection.

Reply:
.. or both. Would this work: "A ConnectionError informs the application that: 1) data could not be delivered after a timeout, or 2) the other side has aborted the Connection."

Reply:
Resolved?

Reply:
Yes, resolved. Thanks!